dig/dig.go

```diff
@@ -120,6 +127,13 @@ func (g *Graph) Resolve(obj interface{}) (err error) {
 	return nil
 }
 
+// MustResolve calls Resolve and panics if an error is encountered
+func (g *Graph) MustResolve(obj interface{}) {
```

Comment:
This is a bit confusing: our current pattern(at least in config) is to have Try methods for no panic behavior and regular verb for methods that can panic.